platform_versioned_docs/version-23.4.0/getting-started/nextflow-getting-started.mdx

```diff
@@ -0,0 +1,216 @@
+---
+title: "Get started with Nextflow"
+description: "An overview and getting started guide for new users of Nextflow."
+date: "14 May 2024"
+tags: [nextflow, getting started, get started]
+---
+
+The Nextflow scripting language is an extension of the Groovy programming language. Nextflow lets you run analysis pipelines on virtually any computing environment. 
+
+Nextflow is a workflow framework that can be used in a wide range of applications to integrate different shell commands into one cohesive pipeline that’s portable, reproducible, scalable, and checkpointed. 
+
+Unless otherwise indicated, processes are run locally. This works well for pipeline development, testing, and small-scale workflows, but for large-scale computational pipelines, a High Performance Cluster (HPC) or Cloud platform is recommended.
```

Comment:
```suggestion
Unless otherwise indicated, processes are run locally. This works well for pipeline development, testing, and small-scale workflows, but for large-scale computational pipelines, a High-Performance Computing (HPC) cluster or cloud platform is recommended.
```
```suggestion
Unless otherwise indicated, processes are run locally. This works well for pipeline development, testing, and small-scale workflows, but for large-scale computational pipelines, a High Performance Cluster (HPC) or Cloud platform is recommended.
```